### PR TITLE
Internal Affairs Remap+Tweaks

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Just some tweaks for IAA and Magistrate; map and job gear.

Map Changes:

- Generally there's two of everything now (stamps, cameras, tape recorders, etc). 
- Adds space law books (why weren't these here before...)
- Adds in a single secure briefcase
- Finally have access to a much needed disposal unit.
- Maintenance access added.
- The door between the magistrate's office and the IAA office is now a glass airlock; to compensate for this more open look, two shutter controls have been added to the magistrate's office, with the second one dropping the shutter ONLY on the door between IA and the judge's very own office.

remap

Related
- IAA starts out with security HUD glasses (check for loyalties amongst sec to ensure SoP is followed)
 - Iconic big styled sunglasses are now in the lawyer's closet.
- Gives Magistrate and IAA bowman headsets 
- Magistrate and IAA both start out with a flash
- Magistrate starts out with a telebaton
- IAA and Magistrate player restriction time increased from 21 to 30 (@Necaladun has suggested this numerous times)

### CODE REVIEW

**Comment by tigercat2000** [comment]
@Fox-McCloud You know what we really need? Less impossible-to-pass-without-an-explosion shutters on doors.

Like, we need polarized doors or something. For this exact purpose.

**Comment by Fox-McCloud** [comment]
Other potential ideas:

- Give IAA's a flash (there's one that can be accessed by them, as is)
- Give the Magistrate a telebaton (he can get stun batons if he wants anyway...)

**Comment by Fox-McCloud** [comment]
Hm, yeah, think I will after looking at Bay's IAA office.

**Comment by Fox-McCloud** [comment]
After taking a look at Bay's IAA office, decided to go ahead with the changes.

**Comment by Necaladun** [comment]
Love a lot of this lots.

The age restriction is great, too many baldies take IAA as "babies first sec role" when it should be played by people who are experienced in sec, not the other way around.

I like the windows - open and transparent, just like our complaints process!

Agree with tiger on the power of those shutters - it'd be great if there was a line between privacy shutters and blast doors. Not a huge issue though.

Having a flash is great, it's the perfect non-lethal defensive rather than offensive weapon. 
The telebaton I'm hesitant on but makes sense mechanically. I'll make especially clear in the magistrate SoP they're not to go hunting with it.

SoP goes the same for IAA - they can also check/update records with it which is great, but I'll make it clear they're not meant to set people on/off arrest.

Bowman headsets I think it'd be better to avoid, paper-pushes should get messed up by flashbangs. Or is there something else they do not aware of?

Name the disposals Filing Cabinet 13 perhaps, for filing stupid complaints?
( https://en.wikipedia.org/wiki/File_13 )


tl;dr love it all muchly except the bowman which really isn't a big issue

**Comment by Aurorablade** [comment]
At least its still handicap accessable for Ssevex..and the maintance door is good incase the hallway is full of plasma. Also can get file stupid complaints and people in filing cabinet 13?

**Comment by Necaladun** [comment]
To file people designated as "stupid" in filing cabinet 13, please request permission from centcomm via fax.

**Comment by Aurorablade** [comment]
does that mean we can have this too? https://github.com/Baystation12/Baystation12/pull/9795 (No i am not being serious...even if it would be fun)

**Comment by Fox-McCloud** [comment]
I mostly included the bowman's for consistency. All of the security force (including those who don't make arrests, set timers, or even have access to flashbangs---like the detective and brig physician) have bowman headsets.

Given that both the magistrate and IAA have sunglasses, it just made sense to give them bowmans so they're not crapped on by their own allies if there's a riot in the brig.

@Aurorablade 

Hahahah. That PR was hilarious.
